### DISCUSSION TITLE
Too hard to hit the button in mania gamemode on a phone

### DISCUSSION THREAD

**Discussion by Invalidcod3** [ppy/osu#24982] [Ideas] [answered]
Playing mania gamemode on the mobile phone without a keyboard is impossible, because the key is too small to hit.
Could osu! add a new setting where the note is flow to the right side, or just make the key bigger (I don't think that could be a solution)

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/23377.
